src/ReactiveUI.Winforms/ContentControlBindingHook.cs

```diff
@@ -17,7 +17,7 @@ namespace ReactiveUI.Winforms
     public class ContentControlBindingHook : IPropertyBindingHook
     {
         /// <inheritdoc/>
-        public bool ExecuteHook(object source, object target, Func<IObservedChange<object, object>[]> getCurrentViewModelProperties, Func<IObservedChange<object, object>[]> getCurrentViewProperties, BindingDirection direction)
+        public bool ExecuteHook(object target, Func<IObservedChange<object, object>[]> getCurrentViewModelProperties, Func<IObservedChange<object, object>[]> getCurrentViewProperties, BindingDirection direction)
```

Comment:
Can you please remove this from this PR and make it a separate PR, doesn't seem to be related to the nullability factor.

Reply:
Sure, can do.  I agree it was a tangential change